### PR TITLE
Set up Books List Add and Remove Features

### PR DESCRIPTION
**Pull request by angeluray**
## Basic Check✔

- Created a new branch according to the Github Flow rules.
- Added a HTML file that includes plain HTML structure for Books List.
- Set up linters for HTML/CSS and JS making sure they work correctly.
- Added a .gitignore file to make sure any irrelevant files are ignored by git.

## Project Requirements Basic checks ✔

- Display all books saved in the collection in the top part of the page. - **Done.**
- Make sure that when a user clicks the "Add" button:
 A new book is added to the collection. - **Done.**

- Make sure that when a user clicks the "Remove" button:
 The correct book is removed from the collection. - **Done.**
- Make sure that data is preserved in the browser's memory by using localStorage - **Done.**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  // Print in the html file
+  printf(collectionBooks);
+  return collectionBooks;
+}
+
+document.querySelector('#bookForm').addEventListener('submit', () => {
+  inputsObject.title = title.value;
+  inputsObject.author = author.value;
+  localStorage.setItem(title.value, JSON.stringify(inputsObject));
+});
```
Comment (rotimiazeez) [inline script.js:51]: - Please make sure data is preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md), reload the page manually to check this after adding a book and removing a book.

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi Team 👏🏽, 
## Good job so far! 👍🏽 
### Summary ♨️ 
- No Linter error ✔️ 
- Good and descriptive PR 💯 
- Correct workflow used 👍🏽 
- Good commit messages 🥇 
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: script.js (hunk 1 above)
Comment (angeluray) [reply script.js:51]: Hi @rotimiazeez, isn't that what I'm doing here?
image

Comment (rotimiazeez) [reply script.js:51]: @angeluray My sincere apologies it was a mix-up from my PC, kindly submit again and it will be approved 🙏🏽

Comment (angeluray) [reply script.js:51]: @rotimiazeez It's ok. I'll submit it again then.

**Review by muneebulrehman — APPROVED** [review]
Hi @angeluray , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Descriptive PR
- Working project

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**